devpi_ldap/main.py

```diff
@@ -95,9 +104,21 @@ def _validate_search_settings(self, configname):
                 fatal("You have to set a 'password' if you use a 'userdn' in LDAP '%s' config." % configname)
 
     def server(self):
-        cfg = self.get('tls', None)
+        warnings.warn("'server()' is deprecated, please use 'server_pool()'.", category=DeprecationWarning, stacklevel=2)
```

Comment:
Why the deprecation warning here? The LDAP object is for internal use in the hooks only.

Reply:
It doesn't hurt, right? 

I didn't want to change the Class interface in a breaking way. The new `server()` still works in places where it was originally used, so I want to notify the users of the change. There may be users of the class that use it differently than originally intended.

So either we remove the `server()` function and simply break stuff that used it. Or we don't break stuff and put the deprecation warning in. Your call.

Reply:
Ok, then just leave it in.